src/components/Navbar.module.css

```diff
@@ -1,26 +1,27 @@
 @import url("https://fonts.googleapis.com/css2?family=Inter:wght@300;600&family=Maven+Pro:wght@500&display=swap");
 
 .logo {
-  font-size: large;
+  /* font-size: large; */
   margin-top: 2px;
   color: blue;
 }
 
 .logoAndName {
   display: flex;
   align-items: center;
-  gap: 10px;
+  gap: 30px;
+  font-size: xx-large;
```

Comment:
`xx-large` and absolute sizes like this are based on the users's default font size (you don't know what that will be)... so will this look good in all situations? If not, it might be best to set a actual size in px, em, rem, etc.

Reply:
I don't see a change (still see `xx-large`) but okay, maybe next PR.